sdk/digitaltwins/dtdl-parser/src/parser/modelParserImpl.ts

```diff
@@ -205,12 +205,10 @@ export class ModelParserImpl implements ModelParser {
         );
       }
 
-      let dependencyMapping: { [dtmi: string]: unknown } | null;
-      try {
-        dependencyMapping = await this.getModels(undefinedIdentifiers);
-      } catch (e) {
-        dependencyMapping = null;
-      }
+      const dependencyMapping = await this.getModels(undefinedIdentifiers, {
+        dependencyResolution: "enabled"
+      });
```

Comment:
2 questions:
1. this catch hid a mistake that I made (getModels not a function). I think we're better without the catch, right?
2. Is this the right dependency resolution mode?

Reply:
can't remember why I implemented it like that with the catch, but yeah as long as it isn't an unhandled promise rejection on failure.